networking/hub-regionA.json

```diff
@@ -415,7 +415,7 @@
                     "name": "[concat(variables('hubFwName'), '/DefaultDnatRuleCollectionGroup')]",
                     "location": "[parameters('location')]",
                     "dependsOn": [
-                        "[variables('hubFwName')]"
+                        "[concat('Microsoft.Network/firewallPolicies/', variables('hubFwName'))]"
```

Comment:
shouldn't use `concat` to do references, needs to be `resourceId` -- this will trip TTK and other validators  Can you reference this (and the others you changed) via the standard `resourceId` function please?

Reply:
after adding
"[resourceId('Microsoft.Network/firewallPolicies', variables('hubFwName'))]",
I got the error 
  "message": "Put on Firewall Policy fw-eastus2 Failed with 1 faulted referenced firewalls"

The recomendation is the simpler name
https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/child-resource-name-type?tabs=json#within-parent-resource

Reply:
Ah! maybe the right thing to do is to name the policy not the same name as the firewall, that'll help keep those two separate, otherwise you have artificially created a 1o1 mapping by name alone.  If you ever applied that to another firewall, it would be "weird" with that name.

Reply:
Good point, I think the fact that it's the FW name really throws it off for me.  If this was the name of the firewallPolicy, that would help readability more.  Sorry for sending you down the wrong path here.

Reply:
Agree, I will change the name. I will create a new name for the policy

Reply:
I will do on Monday

Reply:
Sounds good.  Have a wonderful weekend @v-fearam

Reply:
done